adam/experiment/observer.py

```diff
@@ -281,35 +302,36 @@ def create_logger(params: Parameters) -> "LearningProgressHtmlLogger":
 
         logging_dir = output_dir / experiment_name
         logging_dir.mkdir(parents=True, exist_ok=True)
-        output_html_path = str(logging_dir / "index.html")
+        output_html_path = logging_dir / "index.html"
 
         if include_links_to_images is None:
             include_links_to_images = False
 
         logging.info("Experiment will be logged to %s", output_html_path)
 
-        with open(output_html_path, "w") as outfile:
-            html_dumper = CurriculumToHtmlDumper()
+        html_dumper = CurriculumToHtmlDumper()
+        if not output_html_path.exists():
+            with open(str(output_html_path), "w") as outfile:
 
-            outfile.write(f"<head>\n\t<style>{CSS}\n\t</style>\n</head>")
-            outfile.write(f"\n<body>\n\t<h1>{experiment_name}</h1>")
-            # A JavaScript function to allow toggling perception information
-            outfile.write(
-                """
-                <script>
-                function myFunction(id) {
-                  var x = document.getElementById(id);
-                  if (x.style.display === "none") {
-                    x.style.display = "block";
-                  } else {
-                    x.style.display = "none";
-                  }
-                }
-                </script>
-                """
-            )
+                outfile.write(f"<head>\n\t<style>{CSS}\n\t</style>\n</head>")
+                outfile.write(f"\n<body>\n\t<h1>{experiment_name}</h1>")
+                # A JavaScript function to allow toggling perception information
+                outfile.write(
+                    """
+                    <script>
+                    function myFunction(id) {
+                      var x = document.getElementById(id);
+                      if (x.style.display === "none") {
+                        x.style.display = "block";
+                      } else {
+                        x.style.display = "none";
+                      }
+                    }
+                    </script>
+                    """
+                )
         return LearningProgressHtmlLogger(
-            outfile_dir=output_html_path,
+            outfile_dir=str(output_html_path),
```

Comment:
Should this be a string?

Reply:
Yes, the `LearningProgresssHtmlLogger` is expecting this "output_dir" parameter to be a string -- It's also a full output path and not a dir so its a confusing parameter name. I'll see if I can refactor it without causing too many issues.

Reply:
Did this cause issues?